### DISCUSSION TITLE
border 0 still showing ?

### DISCUSSION THREAD

**Discussion by BastLast** [tailwindlabs/tailwindcss#7492] [Help] [answered]
Hi ! Im fairly new to tailwind and i'm facing an issue with a border-0 still showing.

here is the page : https://canardecarlate.fr

![image](https://user-images.githubusercontent.com/38099970/154316362-d88bad0d-e4f2-489e-9205-e1973beba98c.png)

as you can see there is a border line just above the footer that i did not add and I don't get how i can remove it ?

i saw through the inspector that it was due to this code : 
![image](https://user-images.githubusercontent.com/38099970/154316674-fdadd3e5-dc77-4a17-bf9a-877eb4dd7328.png)

Here is the code : 
https://github.com/Canard-Ecarlate/Ressources/tree/main/docs

**Comment by NovusOpus** [comment, marked as answer]
That border is generated because of `border-b` in the section above your `svg` background:

`<section class="bg-white border-b py-8">`

This translates in normal CSS to:
```CSS
.border-b {
    border-bottom-width: 1px;
}
```

**Reply by BastLast** [reply]
That's it thx !!
